_aggregations/bucket/composite.md

```diff
@@ -0,0 +1,826 @@
+          },
+          "doc_count": 17691
+        },
+        {
+          "key": {
+            "quantity": 1,
+            "unit_price": 50
+          },
+          "doc_count": 5014
+        },
+        {
+          "key": {
+            "quantity": 1,
+            "unit_price": 100
+          },
+          "doc_count": 482
+        },
+        {
+          "key": {
+            "quantity": 1,
+            "unit_price": 150
+          },
+          "doc_count": 148
+        },
+        {
+          "key": {
+            "quantity": 1,
+            "unit_price": 200
+          },
+          "doc_count": 32
+        },
+        {
+          "key": {
+            "quantity": 2,
+            "unit_price": 150
+          },
+          "doc_count": 4
+        }
+      ]
+    }
+  }
+}
+```
+
+The bucket key for each field is the lower bound of the field interval. For example, the `unit_price` key for the first `composite` bucket is `0`.
+
+To retrieve the next `6` buckets, supply the `after` parameter with the `after_key` object from the response as follows:
+
+```json
+GET opensearch_dashboards_sample_data_ecommerce/_search
+{
+  "size": 0,
+  "aggs": {
+    "composite_buckets": {
+      "composite": {
+        "sources": [
+          { "quantity": { "histogram": { "field": "products.quantity", "interval": 1 }}},
+          { "unit_price": { "histogram": { "field": "products.base_unit_price", "interval": 50 }}}
+        ],
+        "size": 6,
+        "after": {
+            "quantity": 2,
+            "unit_price": 150
+        }
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+As shown in the following example response, there are only two buckets left:
+
+```json
+{
+  "took": 12,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4675,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "composite_buckets": {
+      "after_key": {
+        "quantity": 2,
+        "unit_price": 500
+      },
+      "buckets": [
+        {
+          "key": {
+            "quantity": 2,
+            "unit_price": 200
+          },
+          "doc_count": 8
+        },
+        {
+          "key": {
+            "quantity": 2,
+            "unit_price": 500
+          },
+          "doc_count": 4
+        }
+      ]
+    }
+  }
+}
+```
+
+## Date histogram
+
+Use the `date_histogram` to create composite aggregations of date ranges.
+
+OpenSearch represents dates, including `date_interval` bucket keys, as `long` integers representing [milliseconds since epoch](https://en.wikipedia.org/wiki/Unix_time). You can output a date format instead using the `format` parameter. This does not change the key order.
+
+OpenSearch stores date-times in UTC. You can display output results in a different timezone using the `time_zone` parameter.
+
+### Parameters: Date histogram
```

Comment:
This is probably not the best place to add details about Date histogram parameters.
Please consider moving the parameters in details to https://docs.opensearch.org/docs/latest/aggregations/bucket/date-histogram/

Similarly for geotile grid: https://docs.opensearch.org/docs/latest/aggregations/bucket/geotile-grid/

The corresponding documentation should have details explicit about parameters. 
Mentions of these aggregations in composite terms should be just to introduce an example and then point out to the detailed documentation on how those can be customized.